lib/engine/round/g_1846/operating.rb

```diff
@@ -186,34 +191,27 @@ def tile_cost(tile, abilities)
         end
 
         def potential_tiles(hex)
-          return [] if used_teleport(hex) && !connected(hex)
+          return [] if used_token_ability(hex) && !connected_hexes[hex]
```

Comment:
why did you change from connected_hexes?

they should not be able to lay tiles if they don't have a token placed, wouldn't this change allow players to place a tile even if they have not teleported?

Reply:
This should be the same just inlined what connected(hex) was doing.

Reply:
no i don't think so. connected_hexes has been overridden in this class to include teleportation hexes